### DISCUSSION TITLE
Is the SmallRye OpenAPI Gradle Plugin published?

### DISCUSSION THREAD

**Discussion by boetscha** [smallrye/smallrye-open-api#1291] [Q&A] [answered]
I am currently using your OpenAPI maven plugin to generate OpenAPI spec files from my code. I found that you already have an adaption of that plugin for gradle and in the gradle build file you already have the code for publishing this plugin. However, i cannot access it through gradle and it also doesn't appear on plugins.gradle.org

Am I missing something or is the plugin not yet ready to be published? If so, do you have any plans of publishing it soon?

**Comment by MikeEdgar** [comment, marked as answer]
@boetscha we have not published it yet. Hopefully soon with one of the next releases. Have you tried `mvn install` on smallrye-open-api locally and using it that way?

**Reply by boetscha** [reply]
Hi @MikeEdgar thanks for the answer. Yes, I know I can build it locally and use it. However, with multiple people working on the project its more convenient to use a published plugin than having everyone build the plugin locally. Anyway, right now we can still use the maven plugin. As long as that works for us we'll just do it that way until the gradle plugin is published.

The plugin is really helping us out so great work and thanks!

**Reply by codespearhead** [reply]
FYI: [It's now published](https://plugins.gradle.org/plugin/io.smallrye.openapi).

**Comment by Hes-Siemelink** [comment]
Hi there, thanks for creating the gradle plugin. We would also be happy with a published version.

**Comment by Hes-Siemelink** [comment]
Thanks for letting me know!

Hes Siemelink | Principal Product Manager, Digital.ai Release
Digital.ai<http://www.digital.ai/> | Creating a digital world you can trust
 cell: +31 614888571
[signature_1392539882]<https://www.digital.ai/>
CONFIDENTIALITY NOTICE: This e-mail message (including any attachments) is for the sole use of the intended recipient and may contain proprietary, confidential or trade secret information. Any unauthorized review, use, disclosure or distribution is prohibited. If you are not the intended recipient, please contact the sender by reply e-mail and destroy all copies of the original message.


From: Pedro Aguiar ***@***.***>
Date: Saturday, 8 June 2024 at 23:03
To: smallrye/smallrye-open-api ***@***.***>
Cc: Hes Siemelink ***@***.***>, Comment ***@***.***>
Subject: Re: [smallrye/smallrye-open-api] Is the SmallRye OpenAPI Gradle Plugin published? (Discussion #1291)
[EXTERNAL]

FYI: It's now published<https://plugins.gradle.org/plugin/io.smallrye.openapi>.

—
Reply to this email directly, view it on GitHub<https://github.com/smallrye/smallrye-open-api/discussions/1291#discussioncomment-9716269>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AAJNEGBCOKKCLJKUR7YTXOLZGNWQNAVCNFSM6AAAAABJAIVLWGVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TOMJWGI3DS>.
You are receiving this because you commented.Message ID: ***@***.***>

CONFIDENTIALITY NOTICE: This e-mail message (including any attachments) is for the sole use of the intended recipient and may contain proprietary, confidential or trade secret information. Any unauthorized review, use, disclosure or distribution is prohibited. If you are not the intended recipient, please contact the sender by reply e-mail and destroy all copies of the original message.
